Core/Object Arts/Dolphin/MVP/Views/Common Controls/UI.ListView.cls

```diff
@@ -2628,7 +2628,7 @@ onStartup
 
 	NtDll isWine
 		ifTrue: 
-			["Having autoSelectPackages option turned on in System  Browser gives unpleasant scrolling effect when many multiple packages are selected. This also happens in Package Browser. Problem under Wine traced to LVIS_FOCUSED option in ListView>>selectIndex:. Not sure what overall effect it will have turning this off."
+			["Having autoSelectPackages option turned on in System Browser gives unpleasant scrolling effect when many multiple packages are selected. This also happens in Package Browser. Problem under Wine traced to LVIS_FOCUSED option in ListView>>selectIndex:. Not sure what overall effect it will have turning this off."
```

Comment:
When running under Wine the autoSelectPackages option causes an unpleasant scrolling effect when many packages are selected in a System Browser or Package Browser. The problem does not occur without the LVIS_FOCUSED style, the absence of which does mean the loss of focus indication.

Reply:
Much better. I'll stick that into place as is.